src/main/java/org/gridsuite/modification/server/entities/equipment/creation/SubstationCreationEntity.java

```diff
@@ -27,11 +30,18 @@ public class SubstationCreationEntity extends EquipmentCreationEntity {
     @Column(name = "country")
     private Country country;
 
-    public SubstationCreationEntity(String equipmentId, String equipmentName, Country country) {
+    @ElementCollection(fetch = FetchType.EAGER)
```

Comment:
why EAGER ?

Reply:
Because it forgot them when I was on lazy mode.